addons/account/views/res_config_view.xml

```diff
@@ -243,6 +243,6 @@
         </record>
 
         <menuitem id="menu_account_config" name="Settings" parent="menu_finance_configuration"
-            sequence="0" action="action_account_config" groups="base.group_configuration"/>
+            sequence="0" action="action_account_config" groups="base.group_no_one"/>
```

Comment:
Yep. A account manager shouldn't be able neither to install a new app/module nor to change the basic account configuration without the administration rights. This behavior has been applied on all App --> Configuration --> Settings menuitems.